### ISSUE TITLE
UI hangs if chrome's remote-debugging-port is opened in a frame of the node-webkit instance (under debug)

### ISSUE THREAD

**Issue by DinisCruz** [nwjs/nw.js#2717]
I start node-webkit on a local test app using `nodewebkit --remote-debugging-port=9223`, which looks like this:
![image](https://cloud.githubusercontent.com/assets/656739/5238768/8723ed36-78be-11e4-9b9f-555d6a6f38c5.png)
Using the example provided on the top frame, we can open google on it (which works due to the node-webkit frame isolation)
![image](https://cloud.githubusercontent.com/assets/656739/5238774/aa855152-78be-11e4-9bc2-b6a757b4b836.png)
Where it gets interesting is that I can also open the http://localhost:9223 page in there (which is the emote-debugging-port value)
![image](https://cloud.githubusercontent.com/assets/656739/5238780/ee0d16da-78be-11e4-9b76-67d379ecb560.png)
but if I click on the 'Chrome repl' link it opens ok at first, but hangs
![image](https://cloud.githubusercontent.com/assets/656739/5238783/148db332-78bf-11e4-8346-8eaeeef3cf70.png)

I believe I read in one of the node-webkit threads about a similar issue, and I wonder if there is a solution?

Basically what I something like I described in the [Is there a GUI REPL for node-webkit?](https://github.com/rogerwang/node-webkit/issues/2702)issue , which is a window with two frames: 
- top frame with a web-browser
- bottom frame with a repl that has access to the top frame
